Lightyear/Strings.idr

```diff
@@ -117,6 +125,14 @@ squote p = between (char '\'') (char '\'') p
 dquote : Monad m => ParserT m String a -> ParserT m String a
 dquote p = between (char '\"') (char '\"') p
 
+||| Collect the literal string contained between two characters
+literal' : Monad m => Char -> Char -> ParserT m String String
+literal' l r = map pack $ between (char l) (char r) (some (satisfy (/= r)))
+
+||| Literal string between two identical characters
+literal : Monad m => Char -> ParserT m String String
```

Comment:
tbh I think that the name `quoted` overlaps too much with `dquote` and `squote` and obfuscates the 'literal' aspect of the command.